examples/react-simple/src/ix/Transaction.js

```diff
@@ -1,37 +1,162 @@
 import React, {useState} from "react"
 import * as sdk from "@onflow/sdk"
+import * as t from "@onflow/types"
 import {signingFunction} from "./utils/signing-function.js"
+import {authorizationFunction} from "./utils/authorization-function.js"
 
 export const Transaction = () => {
   const [result, setResult] = useState(null)
 
   const run = async () => {
 
-    const acctResponse = await sdk.send(await sdk.pipe(await sdk.build([
-      sdk.getAccount("f8d6e0586b0a20c7")
+    /*
+
+      Transaction
+      -----------
+
+      Transactions are declared by building an interaction and specifying a transaction within it.
+      This is done by specifying Cadence code within the transaction builder.
+
+      Transactions also require other pieces of information as so they can be sent.
+
+      Proposer, Payer and Authorizers
+      --------------------------------
+
+      Transactions require a proposer, payer and an array of authorizers.
+
+      - The proposer specifies the Account for which the sequence number of one of it's keys will be used and incremented by this transaction.
+      - The payer specifies the Account for which will pay for the transaction.
+      - Each authorizer specifies an AuthAccount which can be used within the cadence code of the transaction.
+
+      To declare a proposer, authorizer or payer, you must specify an authorization.
+      The sdk.authorization constructor consumes an account address, a signing function, and a keyId.
+
+      The signing function is a function with an interface as such:
```

Comment:
Is it TMI to add that signing functions are not a part of the SDK and are usually provided by a 3rd party integration, or otherwise must be user-defined?